docs/content/Deployment/Platform-Docker.md

```diff
@@ -0,0 +1,349 @@
+      - CUBEJS_CUBESTORE_HOST=cubestore_router
+      - CUBEJS_REDIS_URL=redis://redis:6379
+      - CUBEJS_API_SECRET=secret
+    volumes:
+      - .:/cube/conf
+    depends_on:
+      - cubestore_worker_1
+      - cubestore_worker_2
+      - cube_refresh_worker
+      - redis
+
+  cube_refresh_worker:
+    image: cubejs/cube:v%CURRENT_VERSION
+    environment:
+      - CUBEJS_DB_TYPE=bigquery
+      - CUBEJS_DB_BQ_PROJECT_ID=cubejs-bq-cluster
+      - CUBEJS_DB_BQ_CREDENTIALS=<BQ-KEY>
+      - CUBEJS_DB_BQ_EXPORT_BUCKET=cubestore
+      - CUBEJS_CUBESTORE_HOST=cubestore_router
+      - CUBEJS_REDIS_URL=redis://redis:6379
+      - CUBEJS_API_SECRET=secret
+      - CUBEJS_SCHEDULED_REFRESH_TIMER=true
+    volumes:
+      - .:/cube/conf
+
+  cubestore_router:
+    image: cubejs/cubestore:v%CURRENT_VERSION
+    environment:
+      - CUBESTORE_WORKERS=cubestore_worker_1:10001,cubestore_worker_2:10002
+      - CUBESTORE_REMOTE_DIR=/cube/data
+      - CUBESTORE_META_PORT=9999
+      - CUBESTORE_SERVER_NAME=cubestore_router:9999
+    volumes:
+      - .cubestore:/cube/data
+
+  cubestore_worker_1:
+    image: cubejs/cubestore:v%CURRENT_VERSION
+    environment:
+      - CUBESTORE_WORKERS=cubestore_worker_1:10001,cubestore_worker_2:10002
+      - CUBESTORE_SERVER_NAME=cubestore_worker_1:10001
+      - CUBESTORE_WORKER_PORT=10001
+      - CUBESTORE_REMOTE_DIR=/cube/data
+      - CUBESTORE_META_ADDR=cubestore_router:9999
+    volumes:
+      - .cubestore:/cube/data
+    depends_on:
+      - cubestore_router
+
+  cubestore_worker_2:
+    image: cubejs/cubestore:v%CURRENT_VERSION
+    environment:
+      - CUBESTORE_WORKERS=cubestore_worker_1:10001,cubestore_worker_2:10002
+      - CUBESTORE_SERVER_NAME=cubestore_worker_2:10002
+      - CUBESTORE_WORKER_PORT=10002
+      - CUBESTORE_REMOTE_DIR=/cube/data
+      - CUBESTORE_META_ADDR=cubestore_router:9999
+    volumes:
+      - .cubestore:/cube/data
+    depends_on:
+      - cubestore_router
+
+  redis:
+    image: bitnami/redis:latest
+    environment:
+      - ALLOW_EMPTY_PASSWORD=yes
+    logging:
+      driver: none
+```
+
+## Security
+
+### Enable HTTPS/TLS
+
+In production, the Cube.js API should be served over a HTTPS connection to
+ensure security of the data in-transit. We recommend using a reverse proxy; as
+an example, let's use [NGINX][link-nginx].
+
+<!-- prettier-ignore-start -->
+[[info |]]
+| You can also use a reverse proxy to enable HTTP 2.0 and GZIP compression
+<!-- prettier-ignore-end -->
+
+First we'll create a new server configuration file called `cube.conf`:
+
+```
+server {
+  listen 443 ssl;
+  server_name cube.my-domain.com;
+
+  ssl_protocols               TLSv1 TLSv1.1 TLSv1.2;
+  ssl_ecdh_curve              secp384r1;
+  ssl_ciphers                 "ECDHE-RSA-AES256-GCM-SHA512:DHE-RSA-AES256-GCM-SHA512:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-SHA384 OLD_TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256 OLD_TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256";
```

Comment:
I don't think that it's a good idea to write ssl_ciphers. There are different formats how it can be specified: like `ALL:!aNULL:!EXPORT56:RC4+RSA:+HIGH:+MEDIUM:+LOW:+SSLv2:+EXP` or `HIGH:!aNULL:!MD5` or by each one with `:` as separator.

Maybe it's better to write a comment above?

Reply:
Can do, I just grabbed them from the LetsEncrypt NGINX example